src/Compilers/Core/Portable/Desktop/AssemblyPortabilityPolicy.cs

```diff
@@ -32,8 +32,7 @@ public override bool Equals(object obj)
 
         public bool Equals(AssemblyPortabilityPolicy other)
         {
-            return this.SuppressSilverlightLibraryAssembliesPortability == other.SuppressSilverlightLibraryAssembliesPortability
-                && this.SuppressSilverlightPlatformAssembliesPortability == other.SuppressSilverlightPlatformAssembliesPortability;
+            return this is { SuppressSilverlightLibraryAssembliesPortability: other.SuppressSilverlightLibraryAssembliesPortability, SuppressSilverlightPlatformAssembliesPortability: other.SuppressSilverlightPlatformAssembliesPortability };
```

Comment:
>other.SuppressSilverlightLibraryAssembliesPortability [](start = 78, length = 53)

Not a valid pattern.

Reply:
➡️ Treating this one as a duplicate of https://github.com/dotnet/roslyn/pull/39738#discussion_r344416577